pkg/systemstatsmonitor/disk_collector.go

```diff
@@ -278,8 +293,9 @@ func (dc *diskCollector) collect() {
 		deviceName := strings.TrimPrefix(partition.Device, "/dev/")
 		fstype := partition.Fstype
 		opttypes := partition.Opts
-		dc.mBytesUsed.Record(map[string]string{deviceNameLabel: deviceName, fsTypeLabel: fstype, mountOptionLabel: opttypes, stateLabel: "free"}, int64(usageStat.Free))
```

Comment:
Could you let me know the reason for stop reporting `mountOptionLabel`?
And since we are essentially changing the definition of an existing metric, maybe we should also document this on the commit message? :P 
It'd be a bit surprising for folks to find out a commit that "adding xxx metric" changed the label of another existing metric lol. WDYT?

Reply:
changed the commit message.